superset/migrations/versions/9a3c5cebcc67_.py

```diff
@@ -0,0 +1,29 @@
+"""empty message
+
+Revision ID: 9a3c5cebcc67
+Revises: 6400c588de5e
+Create Date: 2018-07-18 00:34:03.073314
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '9a3c5cebcc67'
+down_revision = '6400c588de5e'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.alter_column('query', 'start_running_time', nullable=False, new_column_name='time_at_db')
```

Comment:
This table tends to have hanging locks and is hard to alter. I've had issue every time I've had to alter this model where it just completely lock up everything. We'd all need to schedule downtime to allow for this.

Reply:
Maybe we can use some sort of tracing instead.

Reply:
I mean just using `statsd.timing(DURATION)`